src/gt4py/next/program_processors/runners/dace_fieldview/gtir_dataflow.py

```diff
@@ -599,8 +632,17 @@ def _visit_map(self, node: gtir.FunCall) -> DataExpr:
         fun_python_code = gtir_python_codegen.get_source(fun_node)
         tasklet_expression = f"{output_connector} = {fun_python_code}"
 
-        # TODO(edopao): extract offset_dim from the input arguments
-        offset_dim = gtx_common.Dimension("", gtx_common.DimensionKind.LOCAL)
+        input_local_offsets = [
+            input_arg.local_offset for input_arg in input_args if input_arg.local_offset is not None
+        ]
+        if len(input_local_offsets) == 0:
+            raise ValueError(f"Missing information on local dimension for map node {node}.")
```

Comment:
The description indicate that `local_offset` is only set for unstructured.
So what about reductions in fully structured meshes?

Furthermore, as far as I know there is one global reduction, I am not fully sure, but I guess they are over the edges. So how do you handle such cases (where you do not reduce over a local dimension but a vertical/horizontal one?

Reply:
I don't know how that looks like in the IR.. all gt4py unit tests and all icon4py tests pass.